opensrp-reveal/src/test/java/org/smartregister/reveal/util/UtilsTest.java

```diff
@@ -122,6 +126,17 @@ public void testIsResidentialStructureReturnsCorrectValue() throws Exception {
         assertFalse(isResidentialStructure(""));
     }
 
+    @Test
+    public void testGetStructureByName() throws Exception {
+        RevealApplication revealApplication = initRevealApplicationMock();
+        StructureRepository structureRepository = Mockito.mock(StructureRepository.class);
+        when(revealApplication.getStructureRepository()).thenReturn(structureRepository);
+
+        Location location = Mockito.mock(Location.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Mockito.mock' due to existing static import 'org.mockito.Mockito.mock'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5414138)